### ISSUE TITLE
False Audio Output after installing

### ISSUE THREAD

**Issue by seinol** [alexa/alexa-avs-sample-app#765]
My Setup is working correctly, but my setup has the false Audio Output. 
I set the audio output in the "automated-install.sh" script to 3.5mm jack, but after the installation alexa answers over HDMI output and that is false in my case.
Can anyone help me?

I am using Raspbian Jessie Lite (Kernel Version 4.4) with a manually installed LXDE GUI (lxde-core, lxappearance and lightdm Login-Manager) on a Raspberry Pi 3
For Audio Input I am using a USB Soundcard and a USB microphone and this is working as expected.

**Comment by yugoren** [comment]
Hi @seinol,

Could you try forcing the audio output of Pi to 3.5mm jack through `raspi-config`? You can find the setting under Option 8 `Advanced Settings`, under `A6`.

**Comment by seinol** [comment]
@yugoren 
It tried this before without success... also with `amixer cset numid=3 1`

**Comment by dskidmor** [comment]
Hey @seinol, how far have you gone with the Raspbian documentation for [Audio Configuration](https://www.raspberrypi.org/documentation/configuration/audio-config.md)? I see you used `amixer`, but you may have more luck running `sudo raspi-config` or editing the `config.txt`.

**Comment by seinol** [comment]
@dskidmor
Yes, I tried every solution explained in this Repository for Audio issues. Including config.txt, raspi-config, amixer, some GUIs for Audio and also alsaconfig. Nothing worked for me...

**Comment by dskidmor** [comment]
That's quite strange, try verifying we're using `amixer` with the correct value by executing `amixer cget numid=3`. The name of the configuration should be "PCM Playback Route". If it's something incorrect, like "PCM Playback Volume" then try getting other configurations until you find the right one.

Also, what is the current state of your `config.txt`, is it default now or have you forced DVI mode?

**Comment by seinol** [comment]
I stopped solving this issue. I reinstalled the Operating System and configured a new Raspi Alexa and it works now without problems.
Thank you for your help.
BR seinol
